episodes/superspreading-estimate.Rmd

```diff
@@ -144,47 +147,49 @@ epi_contacts %>%
 
 :::::::::::::::::::::::::::
 
-When each infector-infectee row is unique, the number of entries per infector corresponds to the number of secondary cases generated by that individual.
+Our goal is to get the number of secondary cases caused by the observed infected individuals. At the contact data frame, when each infector-infectee pair is unique, the number of rows per infector corresponds to the number of secondary cases generated by that individual.
 
 ```{r}
-# count secondary cases per infector
-infector_secondary <- epi_contacts %>%
+# count secondary cases per infector in contacts
+epi_contacts %>%
   purrr::pluck("contacts") %>%
   dplyr::count(from, name = "secondary_cases")
 ```
 
-But this output only contains number of secondary cases for reported infectors, not for each of the individuals in the whole `epicontacts` object.
+But this output only contains the number of secondary cases for reported infectors in the contact data, not for **all** the individuals in the whole `<epicontacts>` object.
 
-To get this, first, we can use `epicontacts::get_id()` to get the full list of unique identifiers ("id") from the `epicontacts` class object. Second, join it with the count secondary cases per infector stored in the `infector_secondary` object. Third, replace the missing values with `0` to express no report of secondary cases from them.
+Instead, from `{epicontacts}` we can use the function `epicontacts::get_degree()`. The argument `type = "out"` get the **out-degree** of each **node** in the contact network from the `<epicontacts>` class object. In a directed network, the out-degree is the number of outgoing edges (infectees) emanating from a node (infector) ([Nykamp DQ, accessed: 2025](https://mathinsight.org/definition/node_degree)). Also, the argument `only_linelist = TRUE` include individuals in contacts and linelist data frames.
```

Comment:
> Also, the argument `only_linelist = TRUE` include individuals in contacts and linelist data frames.

I think you could be a bit clearer on exactly what this means. Is it only including individuals that are in _both_ the contacts and line list data frames, or only the line list irrespective of the contacts data? And why would these two datasets contain different individuals (i.e. is it more likely that the line list or the contacts data is missing individuals)?

Reply:
Yes this is much clearer, thanks!